buddyboss/buddyboss-platform
# PROD-743 #2906  - Fix: Generating video thumbnail stuck at loading

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2906 .

### How to test the changes in this Pull Request:

1. Make sure the site has ffmpeg installed.
2. Upload video and trying changing thumbnail.
3. Once the thumbnail is generated in the background, see the thumbnail on the pop-up.


### Proof Screenshots or Video
https://vimeo.com/622320805/032f78cbb0


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
